homeassistant/components/home_connect/entity.py

```diff
@@ -17,10 +17,11 @@ class HomeConnectEntity(Entity):
     """Generic Home Connect entity (base class)."""
 
     _attr_should_poll = False
+    _attr_device: HomeConnectDevice
 
     def __init__(self, device: HomeConnectDevice, desc: str) -> None:
         """Initialize the entity."""
-        self.device = device
+        self._attr_device = device
```

Comment:
Should I leave it as `device` and delete the property definition?

Reply:
I think that works yes. The only thing that this setup does is to make it read only to avoid `self.device = ...`, but I don't think there's any significance in doing this here